cpp/ql/src/Critical/OverflowDestination.ql

```diff
@@ -14,7 +14,10 @@
  */
 
 import cpp
-import semmle.code.cpp.security.TaintTracking
+import semmle.code.cpp.ir.dataflow.TaintTracking
+import semmle.code.cpp.controlflow.IRGuards
+import semmle.code.cpp.security.FlowSources
+import DataFlow::PathGraph
```

Comment:
You haven't changed this query to `path-problem`.

Reply:
Doing this should affect the test in `cpp\ql\test\query-tests\Security\CWE\CWE-119\semmle\tests`.

Reply:
Well spotted! Fixed in f2676968f01eca4a1f89f9187fa197dbeb9cd0c4. I had to change add a couple of `getFullyConverted` to remove some duplicated results.